File: src/main/java/com/numbers/discordbot/audio/MusicManagerCache.java
package com.numbers.discordbot.audio;

import com.sedmelluq.discord.lavaplayer.player.*;
import sx.blah.discord.handle.obj.*;

public interface MusicManagerCache {

    GuildMusicManager getGuildMusicManager(IGuild fromGuild);
    
    AudioPlayerManager getAudioPlayerManager();
}

File: src/main/java/com/numbers/discordbot/audio/MusicManagerCache.kt
package com.numbers.discordbot.audio

import com.sedmelluq.discord.lavaplayer.player.AudioPlayerManager
import sx.blah.discord.handle.obj.IGuild

interface MusicManagerCache {

    fun getGuildMusicManager(fromGuild : IGuild) : GuildMusicManager
    val audioPlayerManager : AudioPlayerManager

}